bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment/Atmospheric/Earth/NRLMSISE00.cpp

```diff
@@ -15,21 +16,44 @@ inline void OpenSpaceToolkitPhysicsPy_Environment_Atmospheric_Earth_NRLMSISE00(p
     using ostk::physics::coord::Frame;
     using ostk::physics::coord::spherical::LLA;
     using ostk::physics::env::obj::Celestial;
+    using EarthGravityModel = ostk::physics::environment::gravitational::Earth;
+    // using EarthAtmosphericModel = ostk::physics::environment::atmospheric::Earth;
```

Comment:
```suggestion
```

Reply:
Re-added because I am re-using it for default values